components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -215,10 +219,15 @@ protected void processAuthenticationResponse(HttpServletRequest request,
             if (!StringUtils.isBlank(userInfoFields)) {
                 if (context.getExternalIdP().getIdentityProvider().getClaimConfig() != null && !StringUtils.isBlank
                         (context.getExternalIdP().getIdentityProvider().getClaimConfig().getUserClaimURI())) {
-                    String userClaimUri = context.getExternalIdP().getIdentityProvider().getClaimConfig()
-                            .getUserClaimURI();
-                    if (!Arrays.asList(userInfoFields.split(",")).contains(userClaimUri)) {
+                    ClaimConfig claimConfig = context.getExternalIdP().getIdentityProvider().getClaimConfig();
```

Comment:
it is possible for context.getExternalIdP() to be null.
possible NPE should be handled.
This should be done before the if condition at line 220